kubernetes/stable.yml

```diff
@@ -0,0 +1,48 @@
+kubernetes:
+  base:
+    repoName: webtests/checkout
+    repoOrg: healthcheck
+    imageRepository: 558830342743.dkr.ecr.us-east-1.amazonaws.com
+    environment: stable
+    secretsManager:
+      enabled: true
+      secretsName: stable/checkout-ui-tests/checkout-ui-tests
+    version: latest
+  apps:
+  - name: checkout-tests-cronjob
+    kind: cronjob
+    schedule: "* * * * *"
+    activeDeadlineSeconds: 2400
+    startingDeadlineSeconds: 6000
```

Comment:
Could you explain the rationale behind those numbers?

Reply:
I got them from [here](https://github.com/vtex/io-sre-konfig/blob/master/services/100-cronjobs/004-healthcheck/013-healthcheck-checkout-ui.yml#L9-L12), which is the config for the tests that are running currently